analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/CookieShouldBeHttpOnlyTest.cs

```diff
@@ -68,34 +71,33 @@ public void CookiesShouldBeHttpOnly_WithWebConfigValueSetToTrue(string root)
         public void CookiesShouldBeHttpOnly_WithWebConfigValueSetToFalse(string root)
         {
             var webConfigPath = Path.Combine(root, WebConfig);
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\CookieShouldBeHttpOnly.cs",
-                new CookieShouldBeHttpOnly(AnalyzerConfiguration.AlwaysEnabled),
-                MetadataReferenceFacade.SystemWeb,
-                TestHelper.CreateSonarProjectConfig(root, TestHelper.CreateFilesToAnalyze(root, webConfigPath)));
+            builder.AddPaths("CookieShouldBeHttpOnly.cs")
+                .AddReferences(MetadataReferenceFacade.SystemWeb)
+                .WithSonarProjectConfigPath(TestHelper.CreateSonarProjectConfig(root, TestHelper.CreateFilesToAnalyze(root, webConfigPath)))
+                .Verify();
         }
 
 #else
         [TestMethod]
         public void CookiesShouldBeHttpOnly_NetCore() =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\CookieShouldBeHttpOnly_NetCore.cs",
-                new CookieShouldBeHttpOnly(AnalyzerConfiguration.AlwaysEnabled),
-                GetAdditionalReferences_NetCore());
+            builder.AddPaths("CookieShouldBeHttpOnly_NetCore.cs")
+                .AddReferences(GetAdditionalReferences_NetCore())
+                .Verify();
 
         [TestMethod]
         public void CookiesShouldBeHttpOnly_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Console(
-                @"TestCases\Hotspots\CookieShouldBeHttpOnly.CSharp9.cs",
-                new CookieShouldBeHttpOnly(AnalyzerConfiguration.AlwaysEnabled),
-                GetAdditionalReferences_NetCore().Concat(NuGetMetadataReference.Nancy()));
+            builder.AddPaths("CookieShouldBeHttpOnly.CSharp9.cs")
+                .WithTopLevelStatements()
+                .AddReferences(GetAdditionalReferences_NetCore().Concat(NuGetMetadataReference.Nancy()))
```

Comment:
I this possible? If so, I would preffer to do it this way:
```suggestion
                .AddReferences(GetAdditionalReferences_NetCore())
                .AddReferences(NuGetMetadataReference.Nancy())
```